scripts/ChangesSinceLastRelease.txt

```diff
@@ -0,0 +1,16 @@
+# Adding *PROC, IP6_ADDRESS, EapHostPeerGetDataToUnplumbCredentials back to arch-specific
+Windows.Win32.Foundation.FARPROC removed
+Windows.Win32.Foundation.FARPROC(X64, Arm64) added
+Windows.Win32.Foundation.FARPROC(X86) added
+Windows.Win32.Foundation.NEARPROC removed
+Windows.Win32.Foundation.NEARPROC(X64, Arm64) added
+Windows.Win32.Foundation.NEARPROC(X86) added
+Windows.Win32.Foundation.PROC removed
+Windows.Win32.Foundation.PROC(X64, Arm64) added
+Windows.Win32.Foundation.PROC(X86) added
```

Comment:
I don't think this is right. We represent `FARPROC` for example as an `IntPtr` adorned with `[UnmanagedFunctionPointer(CallingConvention.Winapi)]` to cover all supported archs already.

This will probably break you @AArnott, just a heads up. (https://github.com/microsoft/win32metadata/issues/486)